src/main/java/org/openstreetmap/atlas/utilities/scalars/Angle.java

```diff
@@ -21,6 +21,8 @@
     protected static final int DM7_PER_DEGREE = 10_000_000;
     // There are approximately 57 degrees per radian
     public static final int DM7_PER_RADIAN = 572_957_795;
```

Comment:
Might be confusing for users of this class to have two types of the same measure. Thoughts on replacing `DM7_PER_RADIAN` to be a double and having a single constant? Not sure of the impact right away, but it's only being used in the `radians()` call right now.

Reply:
The int constant is used in a number of different contexts outside the class and the precision loss may not matter there + the refactoring of all int instances to double was outside of my ambition for this PR. This alone will cause refactoring for any precise relying unit tests.